src/ServiceControl/SagaAudit/SagaAuditing.cs

```diff
@@ -112,6 +129,33 @@ public override void Enrich(IReadOnlyDictionary<string, string> headers, IDictio
                 }
 
             }
+
+            static IEnumerable<string> SplitInvokedSagas(string sagasInvokedRaw)
```

Comment:
yeah, if there is more than one entry (which means at least one duplicated) there will always be a semicolon.